backend/integrationtest/api/__snapshots__/hyvaksymisPaatosHyvaksytty.test.ts.snap

```diff
@@ -31,3 +31,24 @@ Object {
   ],
 }
 `;
+
+exports[`Hyväksytyn hyväksymispäätöskuulutuksen jälkeen should get epäaktiivinen and jatkopäätös1 statuses successfully 2`] = `
+Object {
+  "velho.saveProjekti": Array [
+    Array [
+      "1.2.246.578.5.1.2978288874.2711575506",
+      Object {
+        "ensimmainenJatkopaatos": Object {
+          "aktiivinen": true,
```

Comment:
Näissä mocksaveprojektitovelho snapshoteissa pitää lukijan itse sitten tietää, että tuota aktiivinen-kenttää ei lähetetä eteenpäin. 

Toinen havainto on, että itse adapteritestit uusille velho-adaptoinneille käsittelyntilan osalta taitaa puuttua

Reply:
Siellä on yksikkötesti velhoclient.savelle, jonka kautta uudet adaptoinnit testataan. Otin tuon "aktiivinen:true" pois, tuskin sillä on tässä kohtaa merkitystä.